src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp

```diff
@@ -612,8 +612,21 @@ void prepare_primitive_fusing::fuse_simple_primitives(program &p) {
             return true;
         };
 
-        auto reduce_supports_fusings = [](reduce_node& node) -> bool {
+        auto reduce_supports_fusings = [&](reduce_node& node) -> bool {
             auto keep_dims = node.as<reduce>().get_primitive()->keep_dims;
+            auto axes = node.as<reduce>().get_primitive()->axes;
+
+            // oneDNN reduction optimized kernel currently does NOT support eltwise fusing.
+            // So it needs to be forced to block eltwise fusing unless an input tensor is small
```

Comment:
Difficult to understand the comment. I think you omitted that `cldnn is used for small tensor`?

Reply:
Modified
// If reduce tensor size is small, it sets not to fuse eltwise which leads to select oneDNN reference reduction
// Because oneDNN optimized kernel does NOT support eltwise fusing